cypress/e2e/admin/administrateUsers.feature

```diff
@@ -1,69 +1,44 @@
+    Then I can see the user with email '<user_email_edited>' in the table
+    And I enter '<role_to_manage>' email '<user_email>' in search input field
+    Then I can not see user '<user_email>' in the table
 
-  @stable_test
-  Scenario: Deleting a student
-    When I go to administration page
-    And I go to student administration
-    And I enter 'student' email 'alex.abramovic@example.com' in search input field
-    And I click edit student button for 'alex.abramovic@example.com'
-    And I click delete user button to delete user with email 'alex.abramovic@example.com'
+  # admin deletes a student
+    And I go to '<role_to_manage>' administration
+    And I enter '<role_to_manage>' email '<user_email_edited>' in search input field
+    And I click edit '<role_to_manage>' button for '<user_email_edited>'
+    And I click delete user button to delete user with lastname '<user_lastname_edited>'
     And I click on delete button in pop up
-    Then I can not see user 'alex.abramovic@example.com' in the table
-
-  @stable_test
-  Scenario: Adding a new teacher
-    When I go to administration page
-    And I go to teacher administration
-    When I click on FAB
-    When I click on Add Teacher in opened FAB
-    And I fill out the user creation form for 'Karl' 'Müller' with email 'karl.mueller@example.com'
-    And I click on add button to add 'teacher'
-    Then I can see the user with email 'karl.mueller@example.com' in the table
+    And I enter '<role_to_manage>' email '<user_email_edited>' in search input field
+    Then I can not see user '<user_email_edited>' in the table
 
-  @stable_test
-  Scenario: Editing a new teacher
-    When I go to administration page
-    And I go to teacher administration
-    And I enter 'teacher' email 'karl.mueller@example.com' in search input field
-    And I click edit teacher button for 'karl.mueller@example.com'
-    And I change username to 'Theodor' 'Müller-Schmidt'
-    And I change email to 't.mueschmidt@example.com'
-    And I click save changes button
-    Then I can see the user with email 't.mueschmidt@example.com' in the table
-    Then I can not see user 'karl.mueller@example.com' in the table
-
-  @stable_test
-  Scenario: Deleting a teacher
-    When I go to administration page
-    And I go to teacher administration
-    And I enter 'teacher' email 't.mueschmidt@example.com' in search input field
-    And I click edit teacher button for 't.mueschmidt@example.com'
-    And I click delete user button to delete user with email 't.mueschmidt@example.com'
-    And I click on delete button in pop up
-    Then I can not see user 't.mueschmidt@example.com' in the table
\ No newline at end of file
+    Examples:
+      | namespace | admin      | role_to_manage | user_firstname   | user_lastname     | user_email                                  | user_firstname_edited | user_lastname_edited     | user_email_edited                         |
```

Comment:
here the table can be cleaned a bit for the spacing between columns and rows to be aligned.

|-------------|--------------|------------|
|-------------|--------------|------------|